docker/plugins.txt

```diff
@@ -1,5 +1,6 @@
 authorize-project
 basic-branch-build-strategies
+block-queued-job
```

Comment:
since we found the issue to be the config in `tvm.yaml` we don't actually need this, right?

```suggestion
```

Reply:
Not strictly, but it did resolve some errors that were being thrown in the system console.